backend/app/anti_recommenders/open_ai/open_ai_anti_recommender.py

```diff
@@ -22,7 +21,7 @@ def __init__(self) -> None:
                 """
 
     def _create_query(self, record_key: str) -> str:
-        """Create a query with the given wikipedia title for the LLM"""
+        """Create a query with the given Wikipedia title for the LLM"""
 
         return (
             "What are 10 Wikipedia articles on the featured list that are dissimilar but surprisingly similar to the \
```

Comment:
Nit: this will need to be adapted to non-Wikipedia sources.

Reply:
Yes, I agree. `_create_query` and `AntiRecommender.generate_anti_recommendations` now have a required `record_type` parameter.